docs/application/native/guides/text-input/autofill.md

```diff
@@ -0,0 +1,365 @@
+# Autofill
+
+Autofill is a feature that allows you to fill out commonly-entered information such as email, account and address in a text input field. email, account and address in a text input field.
```

Comment:
Please change to:
Autofill is a feature that allows you to fill commonly entered information, such as email, account, and address in a text input field.